src/reducers/account/claims.ts

```diff
@@ -0,0 +1,33 @@
+import { AnyAction } from 'redux'
+import { Map } from 'immutable'
+
+const initialState = Map({
+  loading: false,
+  savedClaims: {
+    claimCategories: ['personal', 'contact'],
```

Comment:
I wonder if there's a need to actually store `claimCategories` here - we can just go through the keys od `savedClaims`, and it makes it easier if we add more in future.